examples/with_winit/src/lib.rs

```diff
@@ -376,9 +377,14 @@ fn run(
                             .surface
                             .get_current_texture()
                             .expect("failed to get surface texture");
+                        // Note: we don't run the async/"robust" pipeline, as
+                        // it requires more async wiring for the readback. See
+                        // [#gpu > async on wasm](https://xi.zulipchat.com/#narrow/stream/197075-gpu/topic/async.20on.20wasm)
+                        #[allow(deprecated)]
+                        // #[expect(deprecated, reason = "This deprecation is not targeted at us.")] // Our MSRV is too low to use `expect`
                         #[cfg(not(target_arch = "wasm32"))]
                         {
-                            scene_complexity = vello::block_on_wgpu(
+                            scene_complexity = vello::util::block_on_wgpu(
```

Comment:
I'd suggest just removing the async pipeline entirely.

Reply:
Would there be a better approach to take entirely?

The examples are copied from vello, but done so a while back. They don't have parity either, some changes were made to play lottie files obviously.

Would it be better to just remove all examples and just have a very simple wasm/native example without all the diagnostics and head scratching?

Reply:
If so, I'd be fine pushing this through and then making a follow up PR to gut the examples with a simpler one.

Reply:
Removing that pipeline doesn't matter that much in the grand scheme of things - this example is more of a demo than an actionable example.